config_tracer.go

```diff
@@ -11,8 +11,9 @@ import (
 
 // TracerConfig configures the OpenTelemetry tracer provider. If not enabled traces will not be sent.
 type TracerConfig struct {
-	Enabled           bool   `yaml:"enabled" bson:"enabled" json:"enabled"`
-	CollectorEndpoint string `yaml:"collector_endpoint" bson:"collector_endpoint" json:"collector_endpoint"`
+	Enabled                   bool   `yaml:"enabled" bson:"enabled" json:"enabled"`
+	CollectorEndpoint         string `yaml:"collector_endpoint" bson:"collector_endpoint" json:"collector_endpoint"`
+	CollectorInternalEndpoint string `yaml:"collector_internal_endpoint" bson:"collector_internal_endpoint" json:"collector_internal_endpoint"`
```

Comment:
I am assuming that this is temporary until we cut over to Kanopy—can we add a TODO here with the JIRA ticket number to remove this field and the conditional logic in the init function once we move to Kanopy?

Reply:
The `CollectorEndpoint` field will still be necessary after the cutover: the agents running in EC2 will still use it.
I'm undecided on the fate of the conditional logic: I'm inclination to leave it so evergreen will still be able to run outside Kanopy if we want it to.